py-polars/tests/unit/test_string_cache.py

```diff
@@ -0,0 +1,122 @@
+from typing import Iterator
+
+import pytest
+
+import polars as pl
+from polars.exceptions import StringCacheMismatchError
+from polars.testing import assert_frame_equal
+
+
+@pytest.fixture(autouse=True)
+def _disable_string_cache() -> Iterator[None]:
+    """Fixture to make sure the string cache is disabled before and after each test."""
+    pl.disable_string_cache()
+    yield
+    pl.disable_string_cache()
+
+
+def sc(set: bool) -> None:
```

Comment:
`sc` is quite cryptic ;)
How about renaming it `assert_string_cache_enabled`

Reply:
I kept it super short to keep the tests legible.

`assert_string_cache_enabled(True)` is not any more legible than `assert pl.using_string_cache is True`